okhttp/src/main/java/okhttp3/internal/platform/Platform.kt

```diff
@@ -150,17 +139,18 @@ open class Platform {
    * should be used specifically for [java.io.Closeable] objects and in conjunction with
    * [logCloseableLeak].
    */
-  open fun getStackTraceForCloseable(closer: String): Any? =
-      when {
```

Comment:
I’m not a fan of big expression functions

Reply:
I'd argue against this one, when is effectively the block and the new form just replaces ` = ` for `{ return`